include/summary-2.tex

```diff
@@ -1,11 +1,11 @@
 A \emph{relation}~$R$ on a set~$X$ is a way of relating elements
 of~$X$. We write $Rxy$ if the relation \emph{holds} between $x$
 and~$y$. Formally, we can consider $R$ as the sets of pairs
-$\tuple{x,y} \in X^2$ such that~$Rxy$.  Being less than, greater than,
-equal to, evenly dividing, being the same length, being a subset of,
```

Comment:
Can you leave in the first "being" (the "evenly dividing" sortof cancels the repetition of "being" if that makes sense).